09a_Custom_Coadd.ipynb

```diff
@@ -0,0 +1,1386 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3bf4a0b0-c88e-4c3c-8525-4e0fd56f91a6",
+   "metadata": {},
+   "source": [
+    "Learn more about the Simple Pipeline Executor."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "e7a8e183-bee2-4117-b830-ca69f78649d1",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# SimplePipelineExecutor.from_pipeline?"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "872efc12-f282-473f-9bf3-fd43eb0a3627",
+   "metadata": {},
+   "source": [
+    "Set up the Simple Pipeline Executor. This step can take up to 5 minutes."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "50c26734-de0e-40f9-899a-42e9de2a52dc",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%time\n",
+    "spe = SimplePipelineExecutor.from_pipeline(assembleCoaddPipeline, where=queryString, butler=simpleButler)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "7fba6582-9f37-46e6-bca0-e4fe673a3fdd",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "Run the pipeline.\n",
+    "\n",
+    "There will be a lot of standard output. Alt-click to the left of the cell and choose \"Enable Scrolling for Outputs\" to condense all of the output into a scrollable inset window.\n",
+    "\n",
+    "This takes about 16 minutes to coadd the 6 visits of Window1."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "ebe78d52-d186-4220-b58a-1dafd518ca0f",
+   "metadata": {
+    "scrolled": true,
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "%%time\n",
+    "quanta = spe.run()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ed37a316-41d3-4bb6-8187-2da1250d71de",
+   "metadata": {},
+   "source": [
+    "# 4. Display and analyse the results\n",
+    "\n",
+    "## 4.1. The quanta\n",
+    "\n",
+    "The \"quanta\" that was just created is a list of objects corresponding to each of the pipeline tasks that were ran.\n",
+    "There will be one for each visit’s makeWarp and one for assembleCoadd.\n",
+    "Each of those has information about, e.g., the name of the task, the reference to the input and output.\n",
+    "\n",
+    "**Option:** learn more about the quanta."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "ed3ae560-1376-41f5-b68c-f27859546c99",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# quanta"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "aeedcf66-5179-46e1-89e8-1cd809eb0aaa",
+   "metadata": {},
+   "source": [
+    "**Option:** to see the pop-up window of methods for quanta, put your cursor after the period and press the tab key."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "9750295b-bcc4-4736-a983-9b24b7713fc8",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# quanta."
```

Comment:
Two things:
1. Instead of having to use the TAB after "quanta.", you can see the methods associated with an object by using "dir" (e.g., `dir(quanta[0])`).
2. Not that I selected `quanta[0]` above -- "quanta" is a list, so the methods you get for it are standard python list methods. What we want to show are things associated with the quanta themselves, which you get by selecting one of the quanta from the list.

Reply:
Added this example of using `dir`.